FabricExample/package.json

```diff
@@ -1,27 +1,41 @@
 {
-  "name": "fabricexample",
+  "name": "FabricExample",
   "version": "0.0.1",
   "private": true,
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "postinstall": "patch-package"
   },
   "dependencies": {
+    "@react-navigation/native": "^6.0.10",
+    "@react-navigation/native-stack": "^6.6.2",
     "react": "18.0.0",
-    "react-native": "0.69.0-rc.0"
+    "react-native": "0.69.0-rc.0",
+    "react-native-gesture-handler": "^2.4.1",
+    "react-native-reanimated": "link:../",
+    "react-native-safe-area-context": "^4.2.5",
+    "react-native-screens": "software-mansion/react-native-screens#568e588817538740dff23a3590748614d4c346e3"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.9",
-    "@babel/runtime": "^7.12.5",
+    "@babel/core": "^7.17.5",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/runtime": "^7.17.2",
     "@react-native-community/eslint-config": "^2.0.0",
+    "@types/jest": "^27.4.1",
+    "@types/react": "^17.0.39",
+    "@types/react-native": "^0.67.1",
+    "@types/react-test-renderer": "^17.0.1",
```

Comment:
Aren't those versions a bit too old for this?

Reply:
Bumped in 494eb1c1b6ac828f03ab8f2904fa90297eac04c3